aleph/manage.py

```diff
@@ -82,7 +85,28 @@ def ensure_collection(foreign_id, label):
     return collection
 
 
-@click.group(cls=FlaskGroup, create_app=create_app)
+def get_aleph_version(ctx: click.Context, _: click.Parameter, value: Any) -> None:
+    if not value or ctx.resilient_parsing:
+        return
+
+    click.echo(
+        f"Aleph {importlib.metadata.version('aleph')}\n"
+        f"Python {platform.python_version()}\n"
+        f"Flask {importlib.metadata.version('flask')}\n"
+        f"Werkzeug {importlib.metadata.version('werkzeug')}\n",
```

Comment:
I know I'm being really pedantic here, but can you remove the trailing newline?

```suggestion
        f"Werkzeug {importlib.metadata.version('werkzeug')}",
```

Reply:
Fair enough. I debated adding that newline in the first place. Fixed in the latest commit.

Reply:
Great!